.github/workflows/mattermost.yml

```diff
@@ -0,0 +1,19 @@
+on:
+  pull_request:
+    types: [ opened, reopened ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    if: ${{ github.actor != 'dependabot[bot]' }}
+    steps:
+      - name: Create the Mattermost Message
+        env:
+          TITLE: ${{ github.event.pull_request.title }}
+          HTML_LINK: ${{ github.event.pull_request.html_url }}
+        run: >
+          jq -n --arg t "Pull request opened: [${TITLE}](${HTML_LINK})" '{text: $t}'
+          > mattermost.json
+      - uses: mattermost/action-mattermost-notify@1.1.0
+        env:
+          MATTERMOST_WEBHOOK_URL: ${{ secrets.MATTERMOST_WEBHOOK_URL }}
```

Comment:
Is this set at org-level or what's the idea for configuring this across repositories?

Reply:
set across wbstack org - I figured that would save us the effort of configuring it everywhere and it's not super secret